ext/url/lib.deno_url.d.ts

```diff
@@ -283,13 +284,14 @@ declare class URLPattern {
   /**
    * Match the given input against the stored pattern.
    *
-   * The input can either be provided as a url string (with an optional base),
-   * or as individual components in the form of an object.
+   * The input can either be provided as an absolute url string,
+   * relative url string with a base url, or as individual components
```

Comment:
```suggestion
   * The input can either be provided as an absolute URL string with an optional base,
   * relative URL string with a required base, or as individual components
```

Added the same suggestion here to not forget about updating it.